src/lib/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestEditor/schema.js

```diff
@@ -139,6 +139,7 @@ export const schema = () => {
       type: {
         title: 'Type',
         type: 'string',
+        default: "PHYSICAL_COPY"
```

Comment:
If this is a vocabulary, we cannot hardcode this value: it can change per instance.